System/Console/ANSI/Unix.hs

```diff
@@ -42,3 +56,45 @@ hHideCursor h = hPutStr h hideCursorCode
 hShowCursor h = hPutStr h showCursorCode
 
 hSetTitle h title = hPutStr h $ setTitleCode title
+
+-- getReportedCursorPosition :: IO String
+-- (See Common-Include.hs for Haddock documentation)
+getReportedCursorPosition = do
+    echo <- hGetEcho stdin -- Preserve existing echo
```

Comment:
Likewise, put this into a bracket.

Reply:
Now bracket-ed.

Reply:
@feuerbach, I apologise for the echoing bug that I introduced on bracketing. Thank you for identifying and fixing it in version 0.7.1.1.

Reply:
No worries :)